### PR TITLE
fix: tests that run unwanted

### PR DESCRIPTION
**Pull request by zale144**
## Description

----

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Updated the scripts for local run, e.g genesis_config_commands.sh if the PR changes parameters
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

----

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: x/rollapp/keeper/grpc_query_rollapp_test.go [hunk 1]
```diff
@@ -15,15 +14,17 @@ import (
...
 	_ = flag.Set("rapid.checks", "50")
 	_ = flag.Set("rapid.steps", "50")
-	rapid.Check(t, func(r *rapid.T) {
-		k, ctx := keepertest.RollappKeeper(t)
+
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
+
+		k, ctx := suite.App.RollappKeeper, suite.Ctx
 		ids := rapid.SampledFrom([]string{
```
Comment (danwt) [inline x/rollapp/keeper/grpc_query_rollapp_test.go:28]: this test is neither long running nor does it run when you run other tests, so no need to change

File: x/rollapp/keeper/liveness_test.go [hunk 2]
```diff
@@ -4,31 +4,33 @@ import (
...
 	rollapps := rapid.StringMatching("^[a-zA-Z0-9]{1,10}$")
 	heights := rapid.Int64Range(0, 10)
 	isJail := rapid.Bool()
-	rapid.Check(t, func(r *rapid.T) {
-		k, ctx := keepertest.RollappKeeper(t)
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
+
+		k, ctx := suite.App.RollappKeeper, suite.Ctx
 		model := make(map[string]types.LivenessEvent) // model actual sdk storage
```
Comment (danwt) [inline x/rollapp/keeper/liveness_test.go:29]: this test is neither long running nor does it run when you run other tests, so no need to change

File: x/rollapp/keeper/liveness_test.go [hunk 3]
```diff
@@ -74,21 +76,18 @@ func TestLivenessEventsStorage(t *testing.T) {
...
+func (suite *RollappTestSuite) TestLivenessFlow() {
 	_ = flag.Set("rapid.checks", "500")
 	_ = flag.Set("rapid.steps", "300")
-	rapid.Check(t, func(r *rapid.T) {
-		s := new(RollappTestSuite)
-		s.SetT(t)
-		s.SetS(s)
-		s.SetupTest()
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
```
Comment (danwt) [inline x/rollapp/keeper/liveness_test.go:84]: prefer change other receivers to `s` 
long receiver names are not go idiomatic

**Review by danwt — CHANGES_REQUESTED** [review]

**Comment by danwt** [comment]
Not sure why you changed the TestLivenessEventsStorage or  TestQuery 
They do not relate to suite and therefore do not run unexpectedly
As we discussed yesterday, right?

File: x/rollapp/keeper/grpc_query_rollapp_test.go (hunk 1 above)
Comment (zale144) [reply x/rollapp/keeper/grpc_query_rollapp_test.go]: done

File: x/rollapp/keeper/liveness_test.go (hunk 2 above)
Comment (zale144) [reply x/rollapp/keeper/liveness_test.go]: done

File: x/rollapp/keeper/liveness_test.go (hunk 3 above)
Comment (zale144) [reply x/rollapp/keeper/liveness_test.go:83]: I prefer `suite`

**Comment by zale144** [comment]
> Not sure why you changed the TestLivenessEventsStorage or TestQuery They do not relate to suite and therefore do not run unexpectedly As we discussed yesterday, right?

It's prettier when they're consistent

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**